service/templates/menu.html

```diff
@@ -76,7 +76,7 @@
             <li><a ng-href="/projects##[[project]]">Projects</a></li>
             {{end}}
             <li><a href="/spawn">Hosts</a></li>
-            <li><a href="/patches/mine">Patches</a></li>
+            <li><a href="{{GetUIBase .User}}/patches/mine">Patches</a></li>
```

Comment:
The new UI has a different URL format. We'll need this to pass the username to the format of http://evergreen.spruce.s3-website-us-east-1.amazonaws.com/#/patches?user=annie.black